src/ansible/deploy.yaml

```diff
@@ -32,36 +35,52 @@
     - name: Print GPU Checking Result
       debug:
         msg: Detected {{ 'NVIDIA' if nvidia_gpu_exist else 'AMD' if amd_gpu_exist else 'N/A' }} GPU
+    - name: Pulling Container Image
+      when: nvidia_gpu_exist | bool and container | bool
+      shell: |
+        docker pull {{ moneo_exporter_nvidia }}
+      become: yes
+      throttle: 32
+    - name: Starting Container
+      when: nvidia_gpu_exist | bool and container | bool
+      shell: |
+        docker rm --force {{ container_name }} && \
+        docker run --name={{ container_name }} --net=host \
+          -e PROFILING={{ profiling }} --rm --runtime=nvidia \
+          --cap-add SYS_ADMIN -v /sys:/hostsys/ -itd {{ moneo_exporter_nvidia }}
+      become: yes
     - name: Copy Source
       synchronize:
         src: ../worker/
         dest: '{{ remote_worker_path }}'
         archive: yes
         recursive: yes
+      when: not container | bool
     - name: Installation
       shell: |
         cd {{ remote_worker_path }}
         bash install/{{ 'nvidia' if nvidia_gpu_exist else 'amd' if amd_gpu_exist else 'common' }}.sh
       become: yes
+      when: not container | bool
     - name: Stop Prior Instances of NVIDIA DCGM Daemon and Exporter
       shell: |
         cd {{ remote_worker_path }} && bash shutdown.sh nvidia
-      when: nvidia_gpu_exist
+      when: nvidia_gpu_exist | bool and not container | bool
       become: yes
     - name: Stop Prior Instances of AMD RDC Daemon and Exporter
       shell: |
         cd {{ remote_worker_path }} && bash shutdown.sh amd
-      when: amd_gpu_exist
+      when: amd_gpu_exist | bool and not container | bool
       become: yes
     - name: Start NVIDIA DCGM Daemon
       shell: |
         nohup nv-hostengine </dev/null >/dev/null 2>&1 &
-      when: nvidia_gpu_exist
+      when: nvidia_gpu_exist | bool and not container | bool
       become: yes
     - name: Start AMD RDC Daemon
       shell: |
         nohup /opt/rocm/rdc/bin/rdcd -u </dev/null >/dev/null 2>&1 &
-      when: amd_gpu_exist
+      when: amd_gpu_exist | bool and not container | bool
```

Comment:
So, the container only supports Nvidia correct? So, in the event a customer has an AMD GPU worker this would attempt to launch the container on it? 
Ideally, if a user chooses the container argument but has AMD GPU worker, the functionality should be that the container argument is ignored.

Can you clarify what would happen in the case of AMD GPU worker and container argument selected?